R/plot.R

```diff
@@ -2,175 +2,298 @@
-  return(net.plot(data.rh, node.scale = 3, edge.scale=1.5, node.lab.cex=label_size, layout.params=order))
+
+
+#' 1c Network plot - number of trials by nodesize and line thickness
+#' 
+#' @param bugsnetdt Long format data.
+#' @param label_size Node label size (default = 1).
+#' @param order Order of nodes (default = NULL).
+#' @return Network plot created by BUGSnet::net.plot().
+make_netplot <- function(bugsnetdt, label_size = 1, order = NULL) {    # added default values and extra option for ordering the nodes (CRN)  
+  data.rh <- BUGSnet::data.prep(arm.data = bugsnetdt, varname.t = "T", varname.s = "Study")
+  return(BUGSnet::net.plot(data.rh, node.scale = 3, edge.scale = 1.5, node.lab.cex = label_size,
+                           layout.params = order
+                           )
+         )
 }
 
-# 1c Creates network connectivity info displayed under network plots 
+
+
+#' 1c Creates network connectivity info displayed under network plots 
+#' 
+#' @param freq List of NMA results created by freq_wrap().
+#' @return Network connectivity created by netmeta::netconnection().
 make_netconnect <- function(freq) {   
   d1 <- freq$d1
-  nc1 <- netconnection(d1$treat1,d1$treat2,d1$studlab, data=NULL)
+  nc1 <- netmeta::netconnection(treat1 = d1$treat1, treat2 = d1$treat2, studLab = d1$studlab, data = NULL)
   print(nc1)
 }
 
-# 2a. Forest Plot
+
+
+#' 2a. Forest Plot
+#' 
+#' @param freq List of NMA results created by freq_wrap().
+#' @param modelranfix "fixed" or "random".
+#' @param ref Reference treatment.
+#' @param min Minimum x-axis limit.
+#' @param max Maximum x-axis limit.
+#' @return Forest plot created by forest.df().
 make_netComp <- function(freq, modelranfix, ref, min, max) {    
-  return(forest.df(freq$net1, modelranfix, freq$lstx, ref, min, max))
+  return(forest.df(netresult = freq$net1, model = modelranfix, lstx = freq$lstx, ref = ref,
+                   min = min, max = max
+                   )
+         )
 }
 
-# 2a. Creates text displayed under forest plots 
+
+
+#' 2a. Creates text displayed under forest plots
+#' 
+#' @param freq List of NMA results created by freq_wrap().
+#' @param outcome_measure "MD", "SMD", "OR", "RR", or "RD".
+#' @param modelranfix "fixed" or "random".
+#' @return Text created by tau.df().
 texttau <- function(freq, outcome_measure, modelranfix){      
-  tau <- round(freq$net1$tau,2)
-  return(tau.df(tau, freq$net1$k, freq$net1$n, modelranfix, outcome_measure))
+  tau <- round(freq$net1$tau, 2)
+  return(tau.df(tau = tau, k = freq$net1$k, n = freq$net1$n, model = modelranfix, outcome = outcome_measure))
 }
 
+
+
+#' Create text to say outcomes are compared to the reference treatment.
+#'
+#' @param ref Reference treatment.
+#' @return Text as above.
 make_refText = function(ref) {
-  y <- paste("All outcomes are versus the reference treatment:", ref)
-  return(y)
+  return(paste("All outcomes are versus the reference treatment:", ref))
```

Comment:
If I am remembering correctly, this should either specify `sep = " "`, or better: 
`paste0("All outcomes are versus the reference treatment: ", ref)`